### PR TITLE
Add staging adjustments

### PR DESCRIPTION
**Pull request by TurnipXenon**
# Overview

Set up a way to have multiple servers up for testing.

## Changes
- Tons of documentation about deploying your own fork plus my workflow
- Add a release script in the Procfile that should migrate the database automatically during deploy (not yet observed tbh)
- Add environment variable overriding in settings.py
- Remove host from required fields in Author, and automatically use base.CURRENT_DOMAIN for the host for local authors

## Testing
My fork: https://github.com/TurnipXenon/cmput404-project-personal-staging
My fork node1: https://turnip-oomfie-1.herokuapp.com/authors/
My fork node 2: https://potato-oomfie.herokuapp.com/authors/

### CODE REVIEW

File: mysocial/mysocial/settings/base.py [hunk 1]
```diff
@@ -152,7 +152,9 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
-
+STATIC_ROOT = os.path.join(BASE_DIR, '/staticfiles/')
```
Comment (TurnipXenon) [inline mysocial/mysocial/settings/base.py:155]: My deploys were not working without this, so I added them back.

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ Before you start:
 Start your virtual env!
 pip install -r requirements.txt
 
+To run locally:
+```bash
+python manage.py runserver --settings mysocial.settings.local
+```
```
Comment (amandanguyeen) [inline README.md:15]: you can double check, but i set up manage.py to run mysocial.settings.local automatically.

File: mysocial/mysocial/settings/base.py (hunk 1 above)
Comment (amandanguyeen) [reply mysocial/mysocial/settings/base.py:155]: oops this was definitely still supposed to be in there

**Review by amandanguyeen — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (TurnipXenon) [reply README.md:15]: I tested this out. I find it weird how it works withou the mysocial.settings.local in `bash for windows` (which behaves more closesly to Unix) but does not work in windows powershell. I'll add a warning in future PRs.

**Pull request merged**